src/ArticleList.jsx

```diff
@@ -0,0 +1,32 @@
+import { useState, useEffect } from "react";
+import axios from 'axios'
+import ArticleCard from './ArticleCard'
+import './ArticleList.css'; 
+
+const ArticleList = () => {
+    const [articles, setArticles] = useState([]);
+    useEffect(() => {
+        axios.get("https://nc-news-be-project-k6p0.onrender.com/api/articles").then((response) => {
+            const sortedArticles = response.data.articles.sort((a, b) => new Date(b.created_at) - new Date(a.created_at))
+            setArticles(sortedArticles);
+        })
+        .catch((error) => {
+            console.log("Error fetching articles:", error);
+        });
+    }, []);
+
+
```

Comment:
Some loading state and some visual feedback for the user whilst data is "loading" should be present.